client/src/components/page/StatsPage.js

```diff
@@ -34,12 +34,17 @@ class StatsPage extends Component {
         return response.json();
       })
       .then(data => {
-        this.setState({ content: data.content });
+        this.setState({ messageCount: data.messageCount });
       });
   }
 
   render() {
-    return <p>{this.state.content}</p>;
+    return (
+      <p>
+        {'The total number of messages: '}
```

Comment:
These braces shouldnt be needed here. Make sure to rerun the code too just to make sure it works without braces.